probables/blooms/countingbloom.py

```diff
@@ -13,7 +13,7 @@
 MISMATCH_MSG = "The parameter second must be of type CountingBloomFilter"
 
 
-def _verify_not_type_mismatch(second):
+def _verify_not_type_mismatch(second: typing.Union[int, "CountingBloomFilter"]) -> bool:
```

Comment:
This  does not need to be a union of int and "CountingBloomFilter". Just "CountingBloomFilter" should be sufficient.

Reply:
This reflects the actual usage. The first PR about typing has been created semi-automatically by using `monkeytype` tool that records what every function actually returns and then examining the changes and refactoring them.

This PR is just rebase of that one with with conflicts solved and some additional refactoring.

Reply:
I think that is from a test to check that it correctly checks for the type and isn't truly allowing int as an input.

Reply:
Fixed.